doc/release/migration_notes.rst

```diff
@@ -274,6 +274,12 @@ With condition::
         ]
     }
 
+.. warning::
+
+    **Update:** It is not recommended to use :ref:`conditions
```

Comment:
I don't see a test or an update to a test for this one. maybe worth adding one to showcase this?

Reply:
Well spotted! I'm preparing an other series on integration tests to cover this point. Should be pushed next week hopefully